awx/main/models/unified_jobs.py

```diff
@@ -1512,3 +1512,35 @@ def log_lifecycle(self, state, blocked_by=None):
         else:
             msg = f"{self._meta.model_name}-{self.id} {state.replace('_', ' ')}"
         logger_job_lifecycle.debug(msg, extra=extra)
+
+    @property
+    def launched_by(self):
+        ancestor_job = self.ancestor_job
+        if ancestor_job.launch_type == "manual" or ancestor_job.launch_type == "relaunch":
+            return ancestor_job.created_by
+        elif ancestor_job.launch_type == "scheduled":
+            return ancestor_job.schedule
+        elif ancestor_job.launch_type == "workflow":
+            return ancestor_job.source_workflow_job
```

Comment:
`source_workflow_job` apparently isn't something provided on the model, it's constructed in the serializer from the `.workflow_job_id`.

Reply:
I don't think this case will ever actually be hit, since by using `.ancestor_job` we are climbing the ladder of workflows until we hit the one that wasn't launched by yet another workflow.  However, in the interest of making sure something weird hasn't happened, this should probably return `ancestor_job.workflow_job`

Reply:
... and it turns out _that_ isn't a field either, it's just calculating from `.get_workflow_job()`.

Reply:
I guess we should just return `ancestor_job.get_workflow_job()` here.